src/System.Windows.Forms.Design/src/System/ComponentModel/Design/ComponentDesigner.cs

```
@@ -626,25 +615,23 @@ protected virtual void PostFilterEvents(IDictionary events)
     /// <summary>
     ///  Allows a designer to filter the set of properties the component it is designing will expose through the TypeDescriptor object.
     /// </summary>
-    protected virtual void PostFilterProperties(IDictionary properties)
+    protected virtual void PostFilterProperties(IDictionary? properties)
```

Why do you have to make it nullable here?

---

I did it because of the null check of `properties` below, but that's probably a poor reason

---

that's the new check you added in this PR. Unless you have a scenario that proves properties can be null, i wouldn't alter functionality.  Let it throw and warn.

---

Okay, I'll change this